src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -46,6 +46,8 @@ export const TiffLoader = {
     for (const prom of promises) {
       // All the errors are logged above so just throw the first error
       if (prom.status === 'rejected') throw new Error('Tiff loading failed: ' + String(prom.reason));
+      // We are processing only 8 bits Tiff for now
+      if ((await is8BitsTiff(prom.value)) === false) throw new Error('Tiff is not a 8 bits TIFF.');
```

Comment:
It looks like we don't need the return value other than to throw this error. In that case we could throw the error inside `is8BitsTiff` and return nothing.